source/_components/ubee.markdown

```diff
@@ -28,8 +28,12 @@ device_tracker:
 ```
 
 {% configuration %}
+model:
+  description: Ubee Router model, e.g. `EVW32C-0N`. If omitted, model will be detected automatically.
+  required: false
+  type: string
```

Comment:
What's the default?

Reply:
Default is to 'detect', to detect model automatically.

Reply:
Let's add that for key `default`.